### PR TITLE
Add generate command

### PR DESCRIPTION
**Pull request by mwierzbicki**
This command generates an example config to given output directory.

### CODE REVIEW

File: cmd/generate.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	"github.com/spf13/cobra"
+)
+
+// generateOptions store render command options
+type generateOptions struct {
+	outDir string
+}
+
+// NewGenerateCmd create new generate command
+func NewGenerateCmd(out io.Writer) *cobra.Command {
```
Comment (mlembke) [inline cmd/generate.go:16]: I am not sure if we want to have such a command. In my opinion config should be automatically generated (if at all) in default location if floodgate is run, there is no config provided and there is no config in default location. There should also be a note for the user that the config was generated in default location and the password field needs to be changed.

Another idea: we should figure out some default location for resources and libraries as well and provide the possibility to pass a parameter with location of resources directory and password to Spinnaker.

However if we decide to keep this command, I think displaying the message for user is still appropriate and there should be an option at least for providing password during generation and default password should be something like `<enter_admin_password_here>`

File: resourcemanager/resourcemanager.go [hunk 2]
```diff
@@ -246,3 +247,20 @@ func (rm ResourceManager) saveResourceData(filePath string, resourceData []byte)
 	}
 	return nil
 }
+
+// SaveStringToFile saves a string to file
+func (rm ResourceManager) SaveStringToFile(filePath string, data string) error {
```
Comment (mlembke) [inline resourcemanager/resourcemanager.go:252]: Maybe this should be moved to `util` package. You could put this in separate file or in `util/utils.go`.

Comment (mwierzbicki) [reply resourcemanager/resourcemanager.go:252]: Good idea

**Comment by mwierzbicki** [comment]
Closing because of no activity. I have no time to finish this PR and if any ideas given in this PR are to be reused, it can be reopened.

**Pull request closed without merging**